rir/src/utils/configurations.h

```diff
@@ -1,39 +1,40 @@
 #ifndef RIR_CONFIGURATIONS_H
 #define RIR_CONFIGURATIONS_H
 
-#include "utils/INIReader.h"
 #include "compiler/translations/pir_translator.h"
+#include "utils/INIReader.h"
 
 namespace rir {
 
 struct Optimization {
-    Optimization(pir::PirTranslator* translator, short order) : translator(translator), order(order) {}
+    Optimization(pir::PirTranslator* translator, short order)
+        : translator(translator), order(order) {}
     ~Optimization() { delete translator; }
-    
+
     pir::PirTranslator* translator;
     short order;
 
-    bool operator < (const Optimization& anotherOptimization) const {
+    bool operator<(const Optimization& anotherOptimization) const {
         if (order == anotherOptimization.order) {
-            return anotherOptimization.translator->getName().compare("cleanup");
+            return anotherOptimization.translator->getName() == "cleanup";
```

Comment:
but this is still not an order. 
`!((1,"a") < (1, "b"))` and `!((1,"b") < (1, "a"))`, but `(1,"a") != (1, "b")`